modules/azure/storage_share/variables.tf

```diff
@@ -19,3 +19,11 @@ variable "enabled_protocol" {
   description = "The protocol used for the share. Possible values are SMB and NFS."
   default     = "SMB"
 }
+
+variable "directories" {
```

Comment:
Should it not be called directories_to_create or something like that?

Reply:
yes, probably you are right